kernels/optimized/cpu/op_mul.cpp

```diff
@@ -130,8 +130,12 @@ Tensor& opt_mul_out(
           out.numel());
     });
   } else if (selected_optimized_path != ElementwiseOptimizedPath::kNone) {
-    auto mul_lambda = [](auto x, auto y) { return x * y; };
-    return torch::executor::handle_broadcast_elementwise(
+    // Reason for using alpha:
+    auto mul_lambda = [](auto x, auto y, auto alpha) {
+      (void)alpha;
```

Comment:
https://en.cppreference.com/w/cpp/language/attributes/maybe_unused

Reply:
thank you :)